services/horizon/internal/expingest/orderbook.go

```diff
@@ -12,7 +13,10 @@ import (
 	"github.com/stellar/go/xdr"
 )
 
-const verificationFrequency = time.Hour
+const (
+	verificationFrequency = time.Hour
+	updateFrequency       = 30 * time.Second
```

Comment:
Can we change it to 5 sec. so it updates with every ledger (on average)?
```suggestion
	updateFrequency       = 5 * time.Second
```